pyface/ui/qt4/workbench/split_tab_widget.py

```diff
@@ -10,6 +10,9 @@
 # Standard library imports.
 import sys
 
+if sys.version_info[0] > 2:
+    unicode = str
```

Comment:
`str` here is being used to coerce the output of `QSplitter.saveState()` which is a `QByteArray`.  From fixing up the TraitsUI code, I found that using `QByteArray`'s `data()` method worked well across versions to give an appropriate Python object (`str` in 2.x, `bytes` in 3.x).